code/acting/src/acting/helper_functions.py

```diff
@@ -30,12 +30,12 @@ def vectors_to_angle_abs(x1: float, y1: float, x2: float, y2: float) -> float:
 
 def vector_angle(x1: float, y1: float) -> float:
     """
-    Returns the angle (radians) between the two given vectors
+    Returns the angle (radians) of a given vectors
```

Comment:
You pass two vectors right? That looks a bit confusiung to me

Reply:
This method returns the angle of a vector relative to the x-axis. Previous commits required a vector for the x-axis and the vector in question. The reference x-axis vector is created in this method and both vectors are then used by `vectors_to_angle_abs` to return the absolute angle between both vectors. `vector_angle` then adds the sign to the vector in order to differentiate between positive and negative heading errors.